edpm_ansible/ansible_plugins/modules/container_config_hash.py

```diff
@@ -209,29 +235,26 @@ def _update_hashes(self):
                 continue
             startup_config_json = json.loads(self._slurp(config))
             config_volumes = self._match_config_volumes(startup_config_json)
-            config_hashes = [
-                self._get_config_hash(vol_path) for vol_path in config_volumes
-            ]
-            config_hashes = filter(None, config_hashes)
-            if 'environment' in startup_config_json:
-                old_config_hash = startup_config_json['environment'].get(
-                    'EDPM_CONFIG_HASH', '')
-            if config_hashes is not None and config_hashes:
-                config_hash = '-'.join(config_hashes)
-                if config_hash == old_config_hash:
+            if config_volumes:
+                if 'environment' in startup_config_json:
+                    old_config_hash = startup_config_json['environment'].get(
+                        'EDPM_CONFIG_HASH', '')
+                    exclude_from_hash = startup_config_json['environment'].get(
+                        'EDPM_EXCLUDE_FROM_HASH', '').split(':')
```

Comment:
I think we better put this out of environment (I mean, add the key to the same level as environment) because this is not really used by container itself.

Reply:
It was my first thought but I kept variables in the container environment as they was. Let me do some tests.

Reply:
ok, now `edpm_config_hash` and `edpm_exclude_from_hash` are top level fields just like `environment`.